timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -5,50 +5,70 @@
 import copy
 import mock
 
+from timesketch.lib.analyzers.sessionizer import SessionizerSketchPlugin
+from timesketch.lib.analyzers.expert_sessionizers import \
+    WebActivitySessionizerSketchPlugin
+from timesketch.lib.testlib import BaseTest
 from timesketch.lib.testlib import MockDataStore
 
 
-class BaseSessionizerTest(object):
-    """Base class for tests for sessionizers."""
+class BaseSessionizerTest(BaseTest):
+    """Tests the base functionality of session analyzers listed in
+    analyzer_classes.
+
+        Attributes:
+            analyzer_classes: A list of analyzer classes to test.
```

Comment:
Extend the docstring to explain that any new analyzers classes should be added here